app/assets/stylesheets/demande.scss

```
@@ -25,3 +25,24 @@
     margin-bottom: 2 * $default-padding;
   }
 }
+
+.champ-content {
+  padding: 0 0 ,5rem;
+
+  p {
+    padding: 0;
+    margin: 0;
+  }
+}
+
+.top-bordered {
+  border-top: 1px solid var(--border-default-grey);
+}
+
+.h-blue-france-500 {
+  color: $blue-france-500;
```

juste en passant, je sais qu'on utilise déjà cette variable scss dans la codebase, mais on devrait toujours utiliser à la place les variable css qui sont contextualisées (type de contenu, contraste, thème…)

```suggestion
  color: var(--text-action-high-blue-france);
```